Taarafo.Core/Migrations/20221117080833_AddGroupMembershipModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Taarafo.Core.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddGroupMembershipModel : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
```

Comment:
When you declare `DbSet<>` in broker files, this migration file will have some code in it. Please, fix this.